sys-kernel/coreos-kernel/coreos-kernel-4.14.26.ebuild

```diff
@@ -19,7 +19,7 @@ DEPEND="${RDEPEND}
 	sys-apps/less
 	sys-apps/sed
 	sys-apps/shadow
-	sys-apps/systemd[cryptsetup]
+	sys-apps/systemd:=[cryptsetup]
```

Comment:
As a learning occasion for me: the effect of this would be to always rebuild in case of slot updates, is that correct?

Reply:
Thanks for pointing this out.  Yes, it rebuilds for slot updates, but I didn't notice the systemd ebuild has a hard-coded `SLOT` value, not `0/${PVR}` like most of our other live ebuilds, so this won't work as intended.  I'll just drop that commit so we stick the the upstream `SLOT` value, and we'll have to remember to bump the kernel revision for systemd upgrades to get it in the initramfs.